journal-temp/original/14-chapter-14.adoc

```diff
@@ -511,25 +555,36 @@ to write two or three lines to the mayor and aldermen of Leeds,
 to cause them to restore the goods to the respective owners.
 +++[+++And thereupon George Whitehead delivered the case in writing to the king.]
 
-King: I will do it; I will speak to my Lord Sunderland to write as you desire.
+[.discourse-part]
+_King:_ I will do it; I will speak to my Lord Sunderland to write as you desire.
 
 +++[+++Which accordingly he did,
```

Comment:
You forgot the [.small-break] here in the original before the line that says [Which accordingly he did, etc.]... though you have it in the modernized. 👍🏻